reactioncommerce/example-storefront
# Remove title tag console warning

Resolves #356 
Impact: minor
Type: bugfix

## Issue
The following warning was present in the console: 
``` 
Warning: <title> should not be used in _document.js's <Head>. https://err.sh/next.js/no-document-title
```

## Solution
Remove the title tag from the `_document.js` component. 

## Testing
1. Verify product grid has a title
2. Verify other pages: `/cart, /checkout, /product/*` have the correct page titles
